### ISSUE TITLE
How to set image to room?

### ISSUE THREAD

**Issue by boozybatsMain** [oxen-io/session-pysogs#224]
I can add token, name, description to room:
`python3 -msogs --add-room token --name "name" --description "desc"`

But how I associate image with that room?

**Comment by BanditUser24** [comment]
In the Tutorial video you were able to upload an image directly in Session  ( Watch 11:30 ) 
https://www.youtube.com/watch?v=5KddQxxwEDg&t=549s

**Comment by Bilb** [comment]
Yes, session-desktop (only desktop I believe) can be used to add an image to a community
